modules/coreos-layering-configuring.adoc

```diff
@@ -33,7 +33,7 @@ For example, the following Containerfile creates a custom layered image from an
 FROM quay.io/openshift-release/ocp-release@sha256... <1>
 #Install hotfix rpm
 RUN rpm-ostree override replace http://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/Packages/kernel-{,core-,modules-,modules-core-,modules-extra-}5.14.0-295.el9.x86_64.rpm && \ <2>
-    rpm-ostree cleanup -m && \
+    dnf clean all && \
```

Comment:
Need to keep using `rpm-ostree cleanup -m` for now until we can switch the `override replace` bit above to `dnf` also.